src/style.css

```
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 * {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 * {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > :first-child {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 > :nth-child(1),
+.stars--2 > :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 > :nth-child(3),
+.stars--2 > :nth-child(4),
+.stars--2 > :nth-child(5) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 > :nth-child(1),
+.stars--3 > :nth-child(2),
+.stars--3 > :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 > :nth-child(4),
+.stars--3 > :nth-child(5) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

```suggestion
.stars--3 > .stars--star:nth-child(-n+3) {
  background-image: url(images/star-active.svg);
}
```
Це все можна спростити, використавши ось такий селектор. Тобто для всіх nth-child по третій включно буде застосовано бекграунд-імедж із зображенням активної зірки.

А для .stars--star по дефолту додай зображення неактивної зірки